Casks/keycue.rb

```diff
@@ -4,12 +4,15 @@
 
   url "https://www.ergonis.com/downloads/products/keycue/KeyCue#{version.no_dots}-Install.dmg",
       user_agent: :fake
-  appcast "https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://www.ergonis.com/downloads/keycue-install.dmg",
-          must_contain: version.no_dots
   name "KeyCue"
   desc "Finds, learns and remembers keyboard shortcuts"
   homepage "https://www.ergonis.com/products/keycue/"
 
+  livecheck do
+    url :homepage
+    regex(/Version.*?(\d+(?:\.\d+)*)/i)
```

Comment:
```suggestion
    url "https://www.ergonis.com/products/keycue/history.html"
    regex(/<h\d>\s*?KeyCue\s+?v?(\d+(?:\.\d+)+)/i)
```

I worry that the looseness of the original regex could lead to it matching something other than what we intend. The latest version is 9.8 but if the page contained some text like "Version 10.0 is on the way!", livecheck would match that as well.

How do you feel about checking the history page (i.e., release information)?